CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## DEV
+
+* Increased maximum allowed version of Cryptography library.
+  Now it requires cryptography < 3.0.
```

Comment:
Maybe clarify even further to `cryptography >= 2, < 3.0` since 1.9 does not work but is less than 3.

Reply:
What's wrong with 1.9?

Reply:
Oh right. I was thinking too big picture on this issue.  1.9 is fine for auth_tkt alone. Nevermind.